agent/Makefile

```diff
@@ -250,10 +250,13 @@ install-lib:
 	${INSTALL} ${INSTALLOPTS} ${LIBDIR}
 	${COPY} lib ${DESTDIR}
 
+# NOTE: we deliberately use `/usr/bin/python3` to make sure that when this is
+# invoked inside a python virtualenv the pip installation won't uninstall the
+# existing pbench module, breaking the environment.
```

Comment:
Is it ever the case that, inside a virtual environment, the version of Python matches `/usr/bin/python3`, such that referencing `/usr/bin/python3` below would _not_ avoid this effect?